packages/hint-axe/src/util/axe.ts

```diff
@@ -137,6 +138,20 @@ const withQuotes = (ruleId: string) => {
 const evaluateAxe = async (context: HintContext, event: CanEvaluateScript, rules: string[]): Promise<AxeResults | null> => {
     const { document, resource } = event;
 
+    /**
+     * iframes scan is ignored for local files due to error:
+     * 'allowedOrigins value "null" is not a valid origin'
+     *
+     * This is caused by an axe-core bug which is currently tracked here:
+     * https://github.com/dequelabs/axe-core/issues/3002
+     */
+    let shouldScanIframes = true;
+    const uri = getAsUri(resource);
+
+    if (uri && uri.protocol.includes('file')) {
+        shouldScanIframes = false;
```

Comment:
nit: The following would allow us to have `shouldScanIframes` as a const:
```
const uri = getAsUri(resource);
const shouldScanIframe = !(uri && uri.protocol.includes('file');
```